wordpress-mobile/WordPress-Aztec-iOS
# Replace gridicons with SFSymbols on the Example app

Refs: #1224

This PR replaces the use of Gridicons on the example app with SFSymbols.

This will remove the only third party dependency of this project and will also allow us to remove the use of Carthage on the build system.

| ![Simulator Screen Shot - iPhone 8 - 2021-01-07 at 14 44 20](https://user-images.githubusercontent.com/651601/104044344-1ceb5200-51d5-11eb-995d-10c44409eb60.png) |  ![Simulator Screen Shot - iPhone 8 - 2021-01-07 at 14 44 57](https://user-images.githubusercontent.com/651601/104044349-1eb51580-51d5-11eb-803b-8507ceaeb0b0.png) |


Notes:
 - This not changes the minimum version supported by the library, Aztec and WordPressEditor are still supporting iOS 11 and above
 - We are only removing the gridicons on Aztec local Example project, the main app will continue to use Gridicons for the legacy editor
 
To test:
 - Check if the project builds correctly locally.
 - Check if the Example app runs and you can use the toolbar to edit
 - Check the overlays on the image using the Image Overlay sample content.
